src/main/java/app/studnicki/ox/Score.java

```diff
@@ -0,0 +1,34 @@
+package app.studnicki.ox;
+
+class Score {
+  int value;
```

Comment:
also can be private ;)

Reply:
I think I will need that field to be at least package private but for now I can make it just private.

Reply:
UPDATE:
However I need it cause I test the value of score.